core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
Okay, I'll kill the file for now and open an issue that says we need a google analytics code block + documentation / example